pkg/cluster/storageaccounts.go

```diff
@@ -5,18 +5,91 @@ package cluster
 
 import (
 	"context"
+	"strings"
 
+	mgmtnetwork "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-08-01/network"
+	"github.com/Azure/go-autorest/autorest/to"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/util/arm"
+	"github.com/Azure/ARO-RP/pkg/util/stringutils"
 )
 
+// enableServiceEndpoints should enable service endpoints on
+// subnets for storage account access
+func (m *manager) enableServiceEndpoints(ctx context.Context) error {
```

Comment:
I don't know about doing this automatically -- I would feel a lot better if this was something we put behind a flag. Maybe not the OperatorFlags, since they are strictly for the operator, but... hmm.

Reply:
Should we have this only work when we enable storageaccounts new logic? If we create private storage accounts, there will be no access without Service Endpoints.